version.txt

```diff
@@ -1 +1 @@
-2.1.1
+2.2.0
```

Comment:
should we use version suffix to differentiate between main & testnet?
e.g. 2.2.0-tesetnet

Reply:
I wouldn't add the testnet prefix/suffix. Server and libs are not adding it.